### PR TITLE
Feature/readme instructions

### PR DESCRIPTION
**Pull request by nice-rain**
- Updated readme to include a more detailed development setup. It gives basic steps to completely setup and run the development site (including setting up a .env file with firebase variables).
- Added a link to the live website for those that may want to visit the current version of the website.

### CODE REVIEW

**Comment by m2mathew** [comment]
Thanks for the submission, @nice-rain. My notifications are now enabled! I will get the team to review this asap!

File: README.md [hunk 1]
```diff
@@ -20,6 +20,25 @@ ReactJS Dallas is a safe place for anyone who wants to help build and develop an
...
 The default Gatsby starter.
 
+## Development Setup
+
+1. Fork and Clone Repository: `git clone --depth=1 https://github.com/<your-username>/reactjs-dallas-site.git`
+2. Setup up repo and install dependencies: `yarn`
+3. Create a new firebase project, in the overview screen click '</>' to find API key and other variables
+4. Create a .env file with the following variables (use the variables from your firebase project):
+```
+GATSBY_FIREBASE_API_KEY=
```
Comment (m2mathew) [inline README.md]: Good idea! Let's also merge #15 to be able to have another way to get the `.env` values into the hands of anyone using this project. 😄

File: README.md [hunk 2]
```diff
@@ -20,6 +20,25 @@ ReactJS Dallas is a safe place for anyone who wants to help build and develop an
 
 The default Gatsby starter.
 
+## Development Setup
+
+1. Fork and Clone Repository: `git clone --depth=1 https://github.com/<your-username>/reactjs-dallas-site.git`
```
Comment (patrickhulce) [inline README.md]: should we de-dupe this with the contributing.md instructions?

Comment (m2mathew) [reply README.md]: Yessir, I'll merge this and update the docs in a subsequent pull request.

**Pull request merged**